src/main/java/swaglabs/common/CommonMethods.java

```
@@ -101,5 +103,31 @@
 		logger.info("Element is added into Cart and Cart Value is " + cartCount);
 		waitForElementContainText(swagLabsHomePage.nonEmptyCartBadge, String.valueOf(cartCount + 1), 10);
 	}
+	
+	public static void loginPageButton(String buttonType) {
+		long l=3000000000l;
+		SwagLabsHomePage swagLabsHomePage = new SwagLabsHomePage(driver);
+		if(buttonType.equalsIgnoreCase("Login")) waitForElementContainText(swagLabsHomePage.logoHeading, "Swag Labs", 10);
+		try {
+			loginPageCancelButton(1);

```

As per Google coding standards, this is incorrect. Empty catch blocks are not recommended. At minimum, log the exception or add a comment explaining why it's safe to ignore. Also, catching generic Exception is not a good practice.